core/src/test/scala/org/broadinstitute/dsde/rawls/expressions/DataRepoExpressionFixture.scala

```diff
@@ -66,8 +96,17 @@ trait DataRepoExpressionFixture {
   // parseable input expressions that are invalid (don't fit the schema, relations, any other reason?)
   val invalidInputExpressions: Seq[String] = Seq(
     "this.column_does_not_exist",
-    "this.case_sample.foo:ref.bar:attribute", // invalid for now -- relations
-    """{"level1": "easy", "other-levels": {"level2": this.gvcf, "level3": [this.library:cohort, "extremely difficult", this.library:cohort.entity]}}""", // invalid -- relations
+    "this.relationship_does_not_exist.second_table_column",
+    "this.rootTableToSecondTable.column_does_not_exist",
+    "this.library:cohort", // namespace:name is not allowed for BQ column/table names
+    "this.library:cohort1",
+    "this.arbitrary:whatever",
```

Comment:
Again, these are only invalid TDR Entity Expressions

Reply:
I'm still concerned about the fact that we added these namespaced expressions to the list of invalid expressions because I think these same fixtures are used in both TDR and Local Entity Expression tests

Reply:
There's a different ExpressionFixture file called LocalExpressionFixture.scala so we can differentiate between what's valid for TDR and what's valid for the local entity provider

Reply:
Maybe I'm missing something, but Intellij tells me that `DataRepoExpressionFixture` is only used by `DataRepoEntityExpressionValidatorSpec` and `LocalEntityLocalExpressionValidatorSpec` uses `LocalExpressionFixture`. Is there something else that I am not considering?